lib/src/model/broadcast/broadcast_preferences.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:convert';
+
+import 'package:flutter/foundation.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:lichess_mobile/src/db/shared_preferences.dart';
+import 'package:lichess_mobile/src/model/auth/auth_session.dart';
+import 'package:lichess_mobile/src/model/common/id.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'broadcast_preferences.freezed.dart';
+part 'broadcast_preferences.g.dart';
+
+const _prefKey = 'broadcast.preferences';
+
+@riverpod
+class BroadcastPreferences extends _$BroadcastPreferences {
```

Comment:
This should be refactored according to the new preferences model. See https://github.com/lichess-org/mobile/blob/8f0e3adce38374fe52180736243b2d135d74e839/lib/src/model/settings/preferences.dart#L15